### PR TITLE
feat: init llvm in spark executor

### LINKED ISSUE #970: Fail when submitting openmldb-batch in Yarn
It throws exception about protobuf when using openmldb-batch in yarn-cluster.

### PR DESCRIPTION
**Pull request by tobegit3hub**
* Resolve https://github.com/4paradigm/OpenMLDB/issues/970 .

### CODE REVIEW

**Review by jingchen2222 — APPROVED** [review]
LGTM

File: java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/RowProjectPlan.scala [hunk 1]
```diff
@@ -115,11 +115,14 @@ object RowProjectPlan {
         val buffer = projectConfig.moduleNoneBroadcast
 
         if (hybridseJsdkLibraryPath.equals("")) {
+          JitManager.initCore()
           JitManager.initJitModule(tag, buffer.getBuffer)
```
Comment (zhanghaohit) [inline java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/RowProjectPlan.scala]: Why put the jit init in RowProjectPlan.scala? It'd better to move to a higher module?

I saw in other *Plan.scala, there are the same init steps for jit.

**Review by zhanghaohit — CHANGES_REQUESTED** [review]
The same problem is also in other places/plan files (e.g., GroupByAggregationPlan.scala and other *Plan.scala)

**Comment by vagetablechicken** [comment]
print this error?https://github.com/4paradigm/OpenMLDB/blob/0fe5ee517976296948df64ab8586ed6978b0eff5/hybridse/src/vm/jit.cc#L156

File: java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/RowProjectPlan.scala (hunk 1 above)
Comment (tobegit3hub) [reply java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/nodes/RowProjectPlan.scala]: You're right. We should add this in other planner or add the method in `initJitModule()`.

**Comment by tobegit3hub** [comment]
> print this error?
> 
> https://github.com/4paradigm/OpenMLDB/blob/0fe5ee517976296948df64ab8586ed6978b0eff5/hybridse/src/vm/jit.cc#L156

This has been added in another pr https://github.com/4paradigm/OpenMLDB/pull/985 .

**Review by vagetablechicken — APPROVED** [review]

**Review by zhanghaohit — CHANGES_REQUESTED** [review]
The two JitManager.initCore definitions can be removed?

It seems that they are never used.

**Review by zhanghaohit — APPROVED** [review]

**Review by aceforeverd — APPROVED** [review]

**Comment by lumianph** [comment]
[bugfix tracking](https://trello.com/c/wEQ9HW0k/25-bugfix-tracking)

**Comment by tobegit3hub** [comment]
[End-to-end offline workflow features](https://trello.com/c/0AErqe3E/32-end-to-end-offline-workflow-features)

**Pull request merged**